Sources/tss-client-swift/Helpers.swift

```diff
@@ -192,6 +192,37 @@ public class TSSHelpers {
 
         return serverCoeffs
     }
+    
+    /// Calculates client(user) coefficients based on the distributed key generation indexes and the user tss index
+    ///
+    /// - Parameters:
+    ///   - patricipatingServerDKGIndexes: The array of indexes for the participating servers.
+    ///   - userTssIndex: The current tss index for the user
+    ///
+    /// - Returns: `String`
+    ///
+    /// - Throws: `TSSClientError`
+    public static func getClientCoefficients(participatingServerDKGIndexes: [BigInt], userTssIndex: BigInt) throws -> String {
```

Comment:
This should be used in tests in addition to just getDKLSCoefficient()

Probably should update the tss-client test to use both this and denormalize share.

Reply:
added getClientCoefficients testcases